Sources/Swift/Tools/ViewCapture/SentryExperimentalMaskRenderer.swift

```diff
@@ -0,0 +1,71 @@
+#if canImport(UIKit) && !SENTRY_NO_UIKIT
+#if os(iOS) || os(tvOS)
+
+import UIKit
+
+class SentryExperimentalMaskRenderer: NSObject, SentryMaskRenderer {
```

Comment:
`m`: Isn't this class almost a duplicate of `SentryDefaultMaskRenderer`? The only difference at a first glance is that this class uses the `SentryGraphicsImageRenderer`. If yes, can't we simply pass in the `SentryGraphicsImageRenderer` instead of the `UIGraphicsImageRenderer` and ditch the duplication?

Reply:
At this point in time, they are mostly equal while using a different renderer. I did consider merging it and only switch the renderer, so we can do that.

But I do not want to change the default implementation which is GA now and make the mask renderer a replaceable/injectable component.

I will re-evaluate to DRY